### ISSUE TITLE
Make server use its own logger but with its own verbose toggle

### ISSUE THREAD

**Issue by EagleoutIce** [Code-Inspect/flowr#271]
After #253

**Comment by EagleoutIce** [comment]
Currently, we mostly need a way to control its verbosity separate

**Comment by EagleoutIce** [comment]
To give more context, i can start flowR with the verbose flag like this:

```shell
$ node dist/src/flowr.js --verbose
```

With this, i get detailed logging output regarding what is happening (and why).

Now, if I start the server with the verbose logging:

```shell
$ node dist/src/flowr.js --server --verbose
```

I get verbose logging for the server output (like 'server listening on port...') but i have no way of controlling these two loggers interdependently. In general, it would be nice to have a way of configuring all loggers within flowR.
